tests/web3js/eth_non_interactive_test.js

```diff
@@ -73,6 +74,29 @@ it('get block', async () => {
     assert.isNull(no)
 })
 
+it('should get block receipts', async () => {
+    let height = await web3.eth.getBlockNumber()
+    assert.equal(height, conf.startBlockHeight)
+
+    let block = await web3.eth.getBlock(height)
+    let response = await helpers.callRPCMethod('eth_getBlockReceipts', [block.hash])
+    assert.equal(response.status, 200)
+
+    let blockReceipts = response.body.result
+    assert.lengthOf(blockReceipts, 3)
+
+    for (let blockReceipt of blockReceipts) {
+        let response = await helpers.callRPCMethod(
+            'eth_getTransactionReceipt',
```

Comment:
Why wouldn't we use web3.js function for this https://docs.web3js.org/guides/web3_eth/methods#gettransactionreceipt

Reply:
Unfortunately, the `web3.js` API does not offer the `eth_getBlockReceipts` endpoint, so we use plain JavaScript for this. And to making comparison easier, I am also using it for `eth_getTransactionReceipt`, to properly compare the JSON result. Because `web3.js` parses hexadecimals to numbers.

Reply:
Updated in https://github.com/onflow/flow-evm-gateway/pull/533/commits/54534f32393401b12961664c1726805b66ceb910